src/Main.java

```diff
@@ -50,51 +50,39 @@ public static void main(String[] args) {
             System.out.println("Если возраст ребёнка равен " + ageChild + " то ему нельзя кататься на аттракционах");
         }
         if (ageChild > 5 && ageChild < 14) {
-            System.out.println("Если возраст ребёнка равен " + ageChild + " можно кататься на аттрационе в совпровождении взросложного");
+            System.out.println("Если возраст ребёнка равен " + ageChild + " можно кататься на аттрационе в сопровождении взросложного");
         }
         if (ageChild > 14) {
             System.out.println("Если возраст ребёнка старше " + ageChild + " можно кататься на аттрационе без сопровождения взрослого");
         }
         // Задача №6
-        int passengerSeat = 103;
-        if (passengerSeat <= 102) {
-            System.out.println("В вагоне есть сидячие или стоячие места ");
-        } else
-            System.out.println("Вагон полностью забит");
+        int full = 102;
+        int seating = 60;
+        int count = 50;
+        if (count > 102) {
```

Comment:
```suggestion
        if (count > full) {
```
нужно использовать переменную, а не числа в коде